### PR TITLE
Cleaned up README.md and changed installation files

### PR DESCRIPTION
**Pull request by Pazdikan**
Hey! I decided to clean up the `README.md` file and I also changed the requirements installation file from `.py` to `.sh` and `.bat` so they would be easier to open for people who don't know much about consoles.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -35,89 +35,54 @@
...
-  - update installed server software to a specific version
-  - supported server software: 
-    - [PaperMc](https://papermc.io/)
-    - [Waterfall](https://papermc.io/downloads#Waterfall)
-    - [Velocity](https://papermc.io/downloads#Velocity)
+# Features
+- Works locally or through SFTP/FTP
+- Runs directly from the console with command line arguments
+- Checks for updates and downloads the latest version of all/specific plugins
+- Checks for updates and downloads the latest version of your server software.
```
Comment (Neocky) [inline README.md:51]: Remove the `.` at the end of the line

**Review by Neocky — CHANGES_REQUESTED** [review]
Just remove the one `.` and other than that this looks really good! Thank you very much for the PR😃

**Comment by Pazdikan** [comment]
Fixed, thanks for noticing! If there's something else to change , let me know 😁

**Comment by Pazdikan** [comment]
My code formatter made changes to some lines, this commit only removes the dot. Sorry for that

**Pull request merged**